src/Storage/custom/hybrid-2019-03-01/Update-AzStorageAccount.ps1

```diff
@@ -0,0 +1,373 @@
+        [Parameter(ParameterSetName='UpdateExpandedKeyVaultEncryption', HelpMessage='Sets the IP ACL rules')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpanded', HelpMessage='Sets the IP ACL rules')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpandedStorageEncryption', HelpMessage='Sets the IP ACL rules')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpandedKeyVaultEncryption', HelpMessage='Sets the IP ACL rules')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Body')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Models.Api20171001.IIPRule[]]
+        # Sets the IP ACL rules
+        ${NetworkRuleSetIPRule},
+    
+        [Parameter(ParameterSetName='UpdateExpanded', HelpMessage='Sets the virtual network rules')]
+        [Parameter(ParameterSetName='UpdateExpandedStorageEncryption', HelpMessage='Sets the virtual network rules')]
+        [Parameter(ParameterSetName='UpdateExpandedKeyVaultEncryption', HelpMessage='Sets the virtual network rules')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpanded', HelpMessage='Sets the virtual network rules')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpandedStorageEncryption', HelpMessage='Sets the virtual network rules')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpandedKeyVaultEncryption', HelpMessage='Sets the virtual network rules')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Body')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Models.Api20171001.IVirtualNetworkRule[]]
+        # Sets the virtual network rules
+        ${NetworkRuleSetVirtualNetworkRule},
+    
+        [Parameter(ParameterSetName='UpdateExpanded', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Parameter(ParameterSetName='UpdateExpandedStorageEncryption', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Parameter(ParameterSetName='UpdateExpandedKeyVaultEncryption', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpanded', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpandedStorageEncryption', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpandedKeyVaultEncryption', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Body')]
+        [System.Management.Automation.SwitchParameter]
+        # A boolean indicating whether or not the service encrypts the data as it is stored.
+        ${QueueEnabled},
```

Comment:
`EncryptQueueService`  This should only be in parameter sets where encryption is enabled